src/cpp/FilesInformation/Loaders/FilesLoader.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by Dmytro Klishch on 10/4/22.
+//
+
+#include "FilesLoader.h"
+#include "../../doctest.h"
+
+Directory *FilesLoader::loadDirectory(const std::string &filepath) {
+    Directory *directory = new Directory(filepath);
+    for (const auto &entry: std::__fs::filesystem::directory_iterator(filepath)) {
```

Comment:
What is std::__fs::filesystem namespace? Standard C++17 uses just std::filesystem. Please fix this - it fails to compile on some platforms